src/Views/EncryptView.vala

```diff
@@ -185,6 +185,15 @@ public class EncryptView : AbstractInstallerView {
             update_next_button ();
         });
 
+        confirm_entry.key_press_event.connect ((event) => {
```

Comment:
There is a native Gtk+ way to do this. You set [`activates_default`](https://valadoc.org/gtk+-3.0/Gtk.Entry.activates_default.html) to true and then set `next_button` as [`has_default`](https://valadoc.org/gtk+-3.0/Gtk.Widget.has_default.html)